### PR TITLE
React docs: code style and small text changes

### PR DESCRIPTION
**Pull request by bulaj**
### Context

Updated code style and small text changes in documentation
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #9737 
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:

- [ ] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [ ] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

[skip changelog]

### CODE REVIEW

File: docs/content/guides/tools-and-building/custom-builds.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ Each Handsontable [project](#about-building) has its own building processes defi
 ## Running your first build
 
 To run your first build:
-1. Install [Node.js](https://nodejs.org/) (version **16.14**+).
-2. Install [npm](https://www.npmjs.com/) (version **8.5**+).
+1. Install [Node.js](https://nodejs.org/) (version **16.15**+).
+2. Install [npm](https://www.npmjs.com/) (version **8.11**+).
```
Comment (warpech) [inline docs/content/guides/tools-and-building/custom-builds.md:79]: Why exactly these versions?

The current versions are Node 16.17.0, npm 8.15.0

Comment (bulaj) [reply docs/content/guides/tools-and-building/custom-builds.md]: updated

**Review by jansiegel — APPROVED** [review]

File: docs/content/guides/tools-and-building/custom-builds.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ Each Handsontable [project](#about-building) has its own building processes defi
 ## Running your first build
 
 To run your first build:
-1. Install [Node.js](https://nodejs.org/) (version **16.14**+).
-2. Install [npm](https://www.npmjs.com/) (version **8.5**+).
+1. Install [Node.js](https://nodejs.org/) (version **16.17**+).
+2. Install [npm](https://www.npmjs.com/) (version **8.15**+).
```
Comment (warpech) [inline docs/content/guides/tools-and-building/custom-builds.md:79]: This is being discussed internally here: https://handsoncode.slack.com/archives/C4PV4S33Q/p1662975119755469

Comment (warpech) [reply docs/content/guides/tools-and-building/custom-builds.md:79]: We decided in the above-linked thread to update the versions before a release, not here. I will revert these lines.

**Review by warpech — APPROVED** [review]
I made some fixes that I perceived as necessary

**Pull request merged**